### PR TITLE
refactor: typehints and return for webrisk samples

### PR DESCRIPTION
**Pull request by Sita04**
## Description

b/280881593 : Fix typehints and return responses from samples to tests.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by kweinmeister** [comment]
LGTM, same [comment](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/9967#pullrequestreview-1428989239) as with other sample re: noxfile @Sita04

**Comment by Sita04** [comment]
Webrisk has an existing noxfile config. Should that be updated?

File: webrisk/snippets/compute_threatlist_diff.py [hunk 1]
```diff
@@ -22,7 +23,7 @@ def compute_threatlist_diff(
     max_diff_entries: int,
     max_database_entries: int,
     compression_type: webrisk_v1.CompressionType,
-) -> None:
+) -> ComputeThreatListDiffResponse:
     """Gets the most recent threat list diffs. These diffs should be applied to a local database of
```
Comment (engelke) [inline webrisk/snippets/compute_threatlist_diff.py:27]: It's preferable to have a single line beginning the docstring, with more details below.

Comment (Sita04) [reply webrisk/snippets/compute_threatlist_diff.py]: Done, thanks!

**Comment by engelke** [comment]
> Webrisk has an existing noxfile config. Should that be updated?

No, you're right. That file already exists, and it already has Enforce Type Hints set to True, so this is fine.

**Review by engelke — APPROVED** [review]
LGTM - thanks! In the future, please note the buganizer number in the PR description.

File: webrisk/snippets/compute_threatlist_diff.py [hunk 2]
```diff
@@ -70,12 +75,12 @@ def compute_threatlist_diff(
...
     # New version token to be used the next time when querying.
     print(response.new_version_token)
-
     # Recommended next diff timestamp.
     print(response.recommended_next_diff)
 
-    print("Obtained threat list diff.")
+    return response
+
+
```
Comment (nicain) [inline webrisk/snippets/compute_threatlist_diff.py:85]: Is this newline necessary for some reason?

Comment (nicain) [reply webrisk/snippets/compute_threatlist_diff.py:85]: I mean, these two newlines :)

File: webrisk/snippets/submit_uri.py [hunk 3]
```diff
@@ -39,6 +43,7 @@ def submit_uri(project_id: str, uri: str) -> None:
     request.submission = submission
 
     response = webrisk_client.create_submission(request)
-    print(f"Submission response: {response}")
+    return response
+
```
Comment (nicain) [inline webrisk/snippets/submit_uri.py:47]: Can this newline be removed?

**Review by nicain — CHANGES_REQUESTED** [review]

**Review by kweinmeister — APPROVED** [review]

File: webrisk/snippets/compute_threatlist_diff.py (hunk 2 above)
Comment (Sita04) [reply webrisk/snippets/compute_threatlist_diff.py:85]: 'black' auto-formatted to include two new lines. Hence, didn't change it.

Should I go ahead and remove the extra new line for all samples?

Comment (engelke) [reply webrisk/snippets/compute_threatlist_diff.py:85]: I wonder if the region tag comment had a blank line after it if black would have added two lines here, or only one. I can't find black rules that explain this specific case.

**Review by nicain — APPROVED** [review]

**Pull request merged**